packages/augur-state/jest.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  "roots": [
+    "<rootDir>/src"
+  ],
+  "transform": {
+    "^.+\\.tsx?$": "ts-jest"
+  },
+  "testRegex": "[\\./]test\\.tsx?$",
+  "moduleFileExtensions": [
+    "ts",
+    "tsx",
+    "js",
+    "jsx",
+    "json",
+    "node"
+  ],
+  "testURL": "http://localhost:8080/"
```

Comment:
Specifying a testURL solves a localStorage issue that appeared after I tried and failed to install ts-jest at the toplevel.